### PR TITLE
Fix prop name value

### PR DESCRIPTION
**Pull request by hectoruch**

### CODE REVIEW

**Comment by hectoruch** [comment]
keyboard

**Comment by sorodrigo** [comment]
Awesome that lib will help us a lot in other places of the app!

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "react-native-hyperlink": "0.0.10",
     "react-native-i18n": "1.0.0",
     "react-native-image-picker": "0.26.3",
+    "react-native-keyboard-aware-scroll-view": "^0.3.0",
     "react-native-keyboard-spacer": "0.3.1",
```
Comment (j8seangel) [inline package.json]: Could we remove this old an unused library?

Comment (hectoruch) [reply package.json]: 👍

**Pull request merged**